hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoTransferHandlerTest.java

```diff
@@ -651,6 +652,71 @@ void failsOnRepeatedAliasAndCorrespondingNumberInTokenTransferList() {
                 .has(responseCode(ACCOUNT_REPEATED_IN_ACCOUNT_AMOUNTS));
     }
 
+    @Test
+    void checkCalculateFeesHandlesExceptionInternally() {
+        // need to set up a transaction that fails because of insufficient payer balance for custom fee
+        config = defaultConfig().getOrCreateConfig();
+        // setup all of the standard stores
+        givenStoresAndConfig(handleContext);
+        // make txn with owner sending 1000 w/ allowances to unknown aliased id
+        // and token transfer of FT from owner 1000 w/ allowances to unknown aliased id
+        // and one NFT from owner to unknown aliased ID
+        givenTxnWithAllowances();
+        ensureAliasesStep = new EnsureAliasesStep(body);
+        replaceAliasesWithIDsInOp = new ReplaceAliasesWithIDsInOp();
+        associateTokenRecepientsStep = new AssociateTokenRecipientsStep(body);
```

Comment:
Can fix in a later PR 😄 

```suggestion
        associateTokenRecipientsStep = new AssociateTokenRecipientsStep(body);
```